src/runtime/internal/keyed-each.ts

```diff
@@ -1,13 +1,17 @@
 import { transition_in, transition_out } from './transitions';
 
+export function remove_block_from_lookup(block, lookup) {
```

Comment:
I'm dubious about the benefits of extracting this out into its own function. It's only called from other helpers, so there are going to be at most two instances of `lookup.delete(block.key)` you're saving.

Also, since this is only called from other helpers, then (if we want to keep it at all) it doesn't need to be exported.

Reply:
Ah, that makes sense. I've gone ahead and removed the function and inlined the call to `lookup.delete(block.key)`.